buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -92,6 +95,11 @@ object Dependencies {
         const val playServicesCastFramework = "com.google.android.gms:play-services-cast-framework:${Versions.playServicesCast}"
     }
 
+    object Car {
+        val mediaMedia = androidx("media", Versions.mediaMedia)
+        var mediaSessionExtension = "com.google.android.exoplayer:extension-mediasession:2.12.0"
```

Comment:
Version isn't aligned with the actual ExoPlayer extension, should use `Versions.exoPlayer`.